src/browser/starter.js

```diff
@@ -17,6 +17,7 @@ import { ScreenAdapter } from "./screen.js";
 import { DummyScreenAdapter } from "./dummy_screen.js";
 import { SerialAdapter, SerialAdapterXtermJS } from "./serial.js";
 import { InBrowserNetworkAdapter } from "./inbrowser_network.js";
+import { SerialAdapterWebSocket } from "./serial_websocket.js";
```

Comment:
Add this file to Makefile:
https://github.com/copy/v86/blob/9735a0eed83a426b3b14cf98f9043a5a9241b142/Makefile#L88-L91
...or merge with [`src/browser/serial.js`](https://github.com/copy/v86/blob/master/src/browser/serial.js)